gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=fba8464465835e74f7270bbf43d6d8a8d7709ab0a43ce1aa3323f73e9aa0c612
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-all.zip
+distributionSha256Sum=c9fefec1c0f508a2aa7365cb3e231110f16a7c478177a354ec2e11637fbaa422
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14-rc-1-all.zip
```

Comment:
Is the demand for Java 24 support urgent enough to justify using a prerelease version of Gradle? Or should we wait until Gradle 8.14 is available?

Reply:
Things seem to work with RC1, but it's not urgent.  https://github.com/uber/NullAway/issues/1189 is blocked on JDK 24 support landing and cutting a new release, but that issue is also not critical.  We can wait until a stable Gradle 8.14 is released.

Reply:
OK with me either way.